modules/ocf_prometheus/manifests/alertmanager.pp

```diff
@@ -16,7 +16,7 @@
   class { '::prometheus::alertmanager':
     version       => '0.15.2',
 
-    extra_options => "--cluster.advertise-address=${::ipaddress}:9094 --web.external-url=\"https://prometheus.ocf.berkeley.edu/alertmanager\"",
+    extra_options => "--cluster.advertise-address=${::ipaddress}:9094 --web.external-url=\"https://dev-prometheus.ocf.berkeley.edu/alertmanager\"",
```

Comment:
this (and similar things in server.pp) should probably just be fixed to actually automatically switch correctly on dev.

Reply:
oh jvperrin already said it lol